src/twisted/conch/ssh/keys.py

```diff
@@ -955,9 +1081,16 @@ def blob(self):
             integer g
             integer y
 
+        EC keys::
+            string 'ecdsa-sha2-[identifier]'
+            integer x
+            integer y
+
+            identifier is the standard NIST curve name
+
         @rtype: L{bytes}
         """
-        type = self.type()
+        type = self.type() #takes care of bad key type.
```

Comment:
Sorry, I forgot to lead with the actionable feedback there: please delete this comment.